browser/js/inject.js

```diff
@@ -42,7 +42,14 @@ const profileInject = {
       wrapper.insertBefore(button, lastButton);
     }
   },
+  "keybasepub": function keybasepubInjectProfile(user) {
```

Comment:
IIRC we decided not to include support for keybase.pub because people can put arbitrary content there and it's just begging for phishing/hijacking.

Reply:
removed :)